itou/www/apply/views/list_views.py

```diff
@@ -431,11 +431,16 @@ def list_for_siae_actions(request):
     can_postpone = all(job_application.postpone.is_available() for job_application in selected_job_applications)
     can_process = all(job_application.process.is_available() for job_application in selected_job_applications)
     can_refuse = all(job_application.refuse.is_available() for job_application in selected_job_applications)
-    can_transfer = (
+    enable_transfer = len(request.organizations) > 1
+    can_transfer = enable_transfer and (
         all(job_application.transfer.is_available() for job_application in selected_job_applications)
-        and len(request.organizations) > 1
     )
-    can_accept = len(selected_job_applications) == 1 and selected_job_applications[0].accept.is_available()
+    cannot_accept_reason = None
+    if len(selected_job_applications) != 1:
+        cannot_accept_reason = "Une seule candidature doit être séléctionnée pour être acceptée."
+    elif not selected_job_applications[0].accept.is_available():
+        cannot_accept_reason = "Cette candidature est déjà acceptée."
```

Comment:
C'est potentiellement sujet à régression si à l'avenir `not accept.is_available()` n'implique plus que `state == ACCEPTED` (jusqu'à très récemment on devait passer par la case à l'étude avant d'accepter).

Soit je resterais sur un message générique "L'état de cette candidature ne permet pas de l'accepter", soit s'assurer d'une manière ou d'une autre que
```set(JobApplicationState) - set(JobApplicationWorkflow.CAN_BE_ACCEPTED_STATES) == {JobApplicationState.ACCEPTED}```

Reply:
Bonne idée, j'ajoute un assert dans le test sur les status possibles